.github/renovate.json

```diff
@@ -31,10 +31,11 @@
     },
     {
       "customType": "regex",
-      "fileMatch": ["^\\.ci\\/env\\/tbb.+$"],
+      "fileMatch": ["^\\.ci\\/env\\/tbb.+$", "^\\.github\\/workflows\\/ci.+$"],
```

Comment:
@icfaust  we might consider moving this to centralized json files in future PRs and point other sources to this one